### PR TITLE
fix: select input "ValueContainer" clickable area

### PR DESCRIPTION
**Pull request by cmonti-bc**
## Description (optional)

nMB7In2nbk

### CODE REVIEW

File: packages/blockchain-info-components/src/Form/SelectInput/template.js [hunk 1]
```diff
@@ -125,6 +125,11 @@ export const sharedSelect = css`
 
 const StyledSelect = styled(Select)`
   ${sharedSelect}
+
+  .dummy-input-wrapper {
+    position: absolute;
+    top: -1000px;
+  }
```
Comment (cmonti-bc) [inline packages/blockchain-info-components/src/Form/SelectInput/template.js:132]: So react-select always adds a `DummyInput` component to the `ValueContainer` but we weren't rendering it so the events that should be passed down/binded to the container where never there. This is for hiding that `DummyInput`.

File: packages/blockchain-info-components/src/Form/SelectInput/template.js [hunk 2]
```diff
@@ -227,6 +238,7 @@ const SelectInput = (props) => {
           Option,
           ValueContainer
         }}
+        inputId={INPUT_ID}
```
Comment (cmonti-bc) [inline packages/blockchain-info-components/src/Form/SelectInput/template.js:241]: image

File: packages/blockchain-info-components/src/Form/SelectInput/template.js (hunk 1 above)
Comment (julianromero-bc) [reply packages/blockchain-info-components/src/Form/SelectInput/template.js:134]: Did you try just using `visibility: hidden;` ?

**Review by julianromero-bc — APPROVED** [review]
if it works I would prefer using the visibility css style

File: packages/blockchain-info-components/src/Form/SelectInput/template.js (hunk 1 above)
Comment (cmonti-bc) [reply packages/blockchain-info-components/src/Form/SelectInput/template.js:134]: https://github.com/JedWatson/react-select/issues/2597

**Review by plondon — COMMENTED** [review]
Any issue w/ other dropdowns? Some to check:

WalletBalanceDropdown on coin view
account dropdown on send crypto screens (legacy, BTC, ETH, XLM, BCH)

**Pull request merged**